test/optimus_snrna/prV3/ValidateOptimus.wdl

```diff
@@ -0,0 +1,272 @@
+workflow ValidateOptimus {
```

Comment:
This one is not in WDL 1.0 like the other 2

Reply:
I removed this test altogether. It was a placeholder that we never replaced